.env

```diff
@@ -1,4 +1,3 @@
 
 VUE_APP_LEDGER_BRIDGE_URL=https://liquality.github.io/ledger-web-bridge/dist
-VUE_APP_ANALYTICS_NAME=wallet-prod
-VUE_APP_ANALYTICS_WRITE_KEY=3jTItMqoo8OuN1gJ6MljKOIsrDjqIZo7
\ No newline at end of file
+VUE_APP_AMPLITUDE_API_KEY=bf12c665d1e64601347a600f1eac729e
```

Comment:
Im not sure if we can remove this from the repo and set as a **secret/env var** in the pipeline before run the npm build task.